### DISCUSSION TITLE
Are there plans to open source the tool for generating documentation?

### DISCUSSION THREAD

**Discussion by lzm0x219** [remix-run/remix#1305] [Q&A] [answered]
I saw that the document was also written by remix, do you have any plans to open source this tool?

**Comment by jangerhofer** [comment, marked as answer]
@kentcdodds commented on this a little while ago. 

 https://twitter.com/kentcdodds/status/1468296671972843521?s=21

I am curious to hear whether his suggestion is still the intent!

**Reply by lzm0x219** [reply]
Thanks.😀
